fineract-provider/src/main/java/org/apache/fineract/cob/api/ConfigureBusinessStepResource.java

```diff
@@ -0,0 +1,122 @@
+import org.apache.fineract.infrastructure.core.serialization.ApiRequestJsonSerializationSettings;
+import org.apache.fineract.infrastructure.core.serialization.DefaultToApiJsonSerializer;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+@Path("/jobs")
+@Component
+@Scope("singleton")
+@Tag(name = "Business Step Configuration", description = "")
+@RequiredArgsConstructor
+public class ConfigureBusinessStepResource {
+
+    private static final Set<String> BUSINESS_STEP_CONFIG_RESPONSE_DATA_PARAMETERS = new HashSet<>(
+            Arrays.asList("jobName", "businessSteps"));
+    private static final Set<String> BUSINESS_STEP_DETAIL_RESPONSE_DATA_PARAMETERS = new HashSet<>(
+            Arrays.asList("jobName", "availableBusinessSteps"));
+
+    private final DefaultToApiJsonSerializer<JobBusinessStepConfigData> businessStepConfigSerializeService;
+    private final DefaultToApiJsonSerializer<JobBusinessStepDetail> businessStepDetailSerializeService;
+    private final ApiRequestParameterHelper apiRequestParameterHelper;
+    private final ConfigJobParameterService configJobParameterService;
+    private final PortfolioCommandSourceWritePlatformService commandWritePlatformService;
+
+    @GET
+    @Path("{jobName}/steps")
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Operation(summary = "List Business Step Configurations for a Job", description = "")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(array = @ArraySchema(schema = @Schema(implementation = ConfigureBusinessStepResourceSwagger.GetBusinessStepConfigResponse.class)))) })
+    public String retrieveAllConfiguredBusinessStep(@Context final UriInfo uriInfo,
+            @PathParam("jobName") @Parameter(description = "jobName") final String jobName) {
+
+        JobBusinessStepConfigData jobBusinessStepConfigData = configJobParameterService.getBusinessStepConfigByJobName(jobName);
+
+        final ApiRequestJsonSerializationSettings settings = apiRequestParameterHelper.process(uriInfo.getQueryParameters());
+        return businessStepConfigSerializeService.serialize(settings, jobBusinessStepConfigData,
+                BUSINESS_STEP_CONFIG_RESPONSE_DATA_PARAMETERS);
+    }
+
+    @PUT
+    @Path("{jobName}/steps")
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Operation(summary = "List Business Step Configurations for a Job", description = "")
+    @ApiResponses({
+            @ApiResponse(responseCode = "204", description = "OK", content = @Content(array = @ArraySchema(schema = @Schema(implementation = ConfigureBusinessStepResourceSwagger.GetBusinessStepConfigResponse.class)))) })
+    public String updateJobBusinessStepConfig(@PathParam("jobName") @Parameter(description = "jobName") final String jobName,
```

Comment:
The update Swagger spec is missing from here hence it won't be generated for the fineract-client project.